src/app/calc/ScouterCalculator.tsx

```diff
@@ -0,0 +1,25 @@
+// フォロワー数の範囲ごとの倍率定義
+const battlePowerMultipliers = [
+  { min: 0, max: 999, multiplier: 10 },
+  { min: 1000, max: 2999, multiplier: 15 },
+  { min: 3000, max: 4999, multiplier: 20 },
+  { min: 5000, max: 7999, multiplier: 25 },
+  { min: 8000, max: 9999, multiplier: 30 },
+  { min: 10000, max: Infinity, multiplier: 50 },
+];
+
+function calculateBattlePower(followers: number): number {
+  if (followers < 0) return 0;
+
+  const range = battlePowerMultipliers.find(
+    (r) => followers >= r.min && followers <= r.max
```

Comment:
[imo]
完全に好みですが、何々以上何々以下を示すときは、こんな感じにすると読みやすい気がしてるんですがどうです？ここら辺誰かと話してみたかったんですよね 笑

イメージこんな感じです
`対象 >= min 対象 <= max`
or
`min <= 対象 <= max`

```suggestion
    (r) => (r.min <= followers && followers <= r.max)
```

Reply:
あー、そう言われると僕もいっちいがおすすめしてくれたほうが見やすいかも！なんか、LeetCodeとかAtCoderだとこういう書き方してること多いから、実際可読性がいいかって言われると、微妙かも。

Reply:
左側の条件式と右側の条件式が反対だと、なんか脳を1回スイッチする感覚あるんですよね〜🤔ﾅｾﾞｶﾊﾜｶﾗﾅｲ